magnetis/astro-native
# [AST-54] Fix FlagIcon

# What

Fixes FlagIcon to match DS definition.
***bonus***: Fix typo in other icons props where `fillRule` was written as `fill-rule` and `clipRule` was written as `clip-rule`

# Sample

| before | after|
|-------|-----|
| <img width="157" alt="Screen Shot 2021-02-23 at 16 41 14" src="https://user-images.githubusercontent.com/8203115/108898416-f2731e00-75f5-11eb-97a9-c256282f118f.png"> | <img width="164" alt="Screen Shot 2021-02-23 at 16 40 47" src="https://user-images.githubusercontent.com/8203115/108898365-e25b3e80-75f5-11eb-9a4a-9b845d154bd0.png"> |



# QA
1. Checkout to branch and run storybook;
2. Select FlagIcon on storybook;
3. Check that corresponds to lastest DS definition:
![Screen Shot 2021-02-23 at 16 45 46](https://user-images.githubusercontent.com/8203115/108898939-965cc980-75f6-11eb-9e4a-7da78100113f.png)











[AST-54](https://produtomagnetis.atlassian.net/browse/AST-54)
